### ISSUE TITLE
Research and Evaluate Pre-trained Models

### ISSUE THREAD

**Issue by boldenj2** [B-U-I-L-T-UIUC/AI-Vision-Adaptive-Face-Masking#3]
Look at models for:
- Face detection and tracking (e.g., OpenCV, Dlib, or MediaPipe)
- Image segmentation or style transfer (e.g., TensorFlow, PyTorch, Hugging Face models)

Evaluate the models:
- Test models on various faces under different lighting conditions.
- Compare performance and accuracy between models.
- Compare how models respond to different lighting and background people
- How do the models handle partial occlusions (hats, glasses, religious headwear, facial hair, etc.)-
